app/views/layouts/application.html.erb

```diff
@@ -21,6 +21,15 @@
       </div>
     <% end %>
 
+    <% if @content_item.recruitment_survey_url %>
+      <%= render "govuk_publishing_components/components/intervention", {
```

Comment:
I would be tempted to wrap the intervention component in a div here that uses a Design System spacing class to add some space between the intervention component and main header, for example:

```erb
    <% if @content_item.recruitment_survey_url %>
      <div class="govuk-!-static-margin-top-4">
        <%= render "govuk_publishing_components/components/intervention", {
          suggestion_text: "Help improve a new GOV.UK tool",
          suggestion_link_text: "Sign up to take part in user research",
          suggestion_link_url: @content_item.recruitment_survey_url,
          new_tab: true,
        } %>
      </div>
    <% end %>

Using this approach, we can then remove the custom `intervention.scss` component stylesheet and related import.
```

Reply:
Great suggestion! Thank you 🙂